test/runtime/Policies.go

```diff
@@ -951,4 +952,135 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 		Expect(strings.TrimSpace(policyMapsInVM.GetStdOut())).To(Equal(expected), "Only %s PolicyMap should be present", expected)
 
 	})
+
+	It("checks policy trace output", func() {
+
+		httpd2Label := "id.httpd2"
```

Comment:
Instead of using `tests/12-policy-import.sh` policy and containers labelled `id.foo` and `id.bar`, I used containers with labels added in the `SampleContainersActions` helper function that is used in the `BeforeEach` in this test.

Reply:
Furthermore, I didn't see the point of using the `teamA` labels in the Bash test because they weren't really used for anything as far as I can tell.